.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+[STORY_LINK]
+
+## Why?
+
+Describe why you do this PR.
+
+## Changes
+
+What are the changes. Explain them explicitly.
+
+## Screenshot
```

Comment:
Since there can be 1 or more screenshots. Should we write "Screenshot(s)" instead?
```suggestion
## Screenshot(s)
```